### PR TITLE
feat: add `publick8s` cluster

### PR DESCRIPTION
**Pull request by lemeurherve**

### CODE REVIEW

File: config/ext_public-nginx-ingress_publick8s.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+controller:
+  replicaCount: 2
+  service:
+    annotations:
+      service.beta.kubernetes.io/azure-load-balancer-internal: false
+    loadBalancerIP: 52.167.253.43
```
Comment (lemeurherve) [inline config/ext_public-nginx-ingress_publick8s.yaml:6]: To be replaced by the correct value

File: config/ext_private-nginx-ingress_publick8s.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+controller:
+  replicaCount: 2
+  ingressClass: private-ingress
+  service:
+    annotations:
+      service.beta.kubernetes.io/azure-load-balancer-internal-subnet: data-tier
```
Comment (lemeurherve) [inline config/ext_private-nginx-ingress_publick8s.yaml:6]: This answers my previous question about the creation of the corresponding subnet in azure-net

File: config/ext_datadog_prodpublick8s.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+datadog:
+  clusterName: 'prodpublick8s'
+  kubelet:
+    host:
+      valueFrom:
+        fieldRef:
+          fieldPath: spec.nodeName
+    hostCAPath: /etc/kubernetes/certs/kubeletserver.crt
+    tlsVerify: false # Required as of Agent 7.35 because Kubelet certificates in AKS do not have a Subject Alternative Name (SAN) set.
```
Comment (lemeurherve) [inline config/ext_datadog_prodpublick8s.yaml:9]: To be revised?

Comment (dduportal) [reply config/ext_datadog_prodpublick8s.yaml:9]: Did AKS started to use a SAN for the kubelet's generated certificate ?

File: config/ext_private-nginx-ingress_publick8s.yaml (hunk 2 above)
Comment (dduportal) [reply config/ext_private-nginx-ingress_publick8s.yaml:6]: Good catch. It meanse that this network must exist in order to have the internal LB created within.

File: config/ext_public-nginx-ingress_publick8s.yaml (hunk 1 above)
Comment (dduportal) [reply config/ext_public-nginx-ingress_publick8s.yaml:6]: - ✅ for the Ipv4 (value should come from a predefined, in terraform, LB)

Comment (dduportal) [reply config/ext_public-nginx-ingress_publick8s.yaml:6]: Also, we'll need to add a 2nd Service of type LB (might need a "raw" YAML unless supported by the ingress-nginx chart): https://stackoverflow.com/a/61898643

**Comment by lemeurherve** [comment]
Everything from this PR has been deployed via other PRs, closing.

**Pull request closed without merging**